src/Magnum/Math/Geometry/Intersection.h

```diff
@@ -188,6 +201,89 @@ template<class T> bool boxFrustum(const Range3D<T>& box, const Frustum<T>& frust
     return true;
 }
 
+template<class T> bool sphereFrustum(const Vector3<T>& center, const T radius, const Frustum<T>& frustum) {
+    const T radiusSq = radius*radius;
+
+    for(const Vector4<T>& plane: frustum.planes()) {
+        /* The sphere is in front of one of the frustum planes (normals point
+           outwards) */
+        if(Distance::pointPlaneScaled<T>(center, plane) < -radiusSq)
+            return false;
+    }
+
+    return true;
+}
+
+/**
```

Comment:
Could the functions below be moved up in front of the long function bodies to be together with the documented function declarations? Reason: people reading docs straight from header files would not need to scroll this far down.